docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc

```diff
@@ -29,7 +29,10 @@ endif::[]
 ifeval::["{build}" == "downstream"]
 SOURCE_DB_ROOT_PASSWORD=$(cat ~/overcloud-deploy/overcloud/overcloud-passwords.yaml | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
 endif::[]
+endif::[]
+ifeval::["{OpenStackPreviousInstaller}" == "director_operator"]
 MARIADB_CLIENT_ANNOTATIONS='--annotations=k8s.v1.cni.cncf.io/networks=internalapi'
```

Comment:
I think those annotations are for non-ospdo as well. (?)

Reply:
it doesn't look like so? see the password etc evaluated below